playbook/app/pb_kits/playbook/packs/site_styles/_landing-page.scss

```diff
@@ -9,6 +9,7 @@
     background-color: $bg_dark;
     border-radius: $border_rad_heaviest;
     box-shadow: $shadow_deepest;
+    min-width: 733px;
```

Comment:
Curious on the specific number `733px` here. Could we possibly make use of [screen size tokens](https://github.com/powerhome/playbook/blob/master/playbook/app/pb_kits/playbook/tokens/_screen_sizes.scss#L5)?

Reply:
733px is the smallest width that lined the black edge of the card up with the avatar card above it, without changing the original design.

Reply:
I can show you what it would look like with that token.

Reply:
https://share.getcloudapp.com/ApuGJ01k @thestephenmarshall

Reply:
I see. Ok just figured I would ask!

Reply:
Of course. I was hoping one would! I don't like not using them for sure. 100%.